Routers/post-router.js

```diff
@@ -0,0 +1,108 @@
+const express = require('express')
+
+const postDB = require('../data/helpers/postDb')
+
+const router = express.Router();
+
+
+
+router.get('/', (req, res) => {
+postDB
+    .get()
+    .then(post => {
+        res.status(200).json(post)
+    })
+    .catch(error => {
+        res.status(400).json({
+            error: "couldnt get posts"
+        })
+    })
+
+})
+
+
+
+router.get('/:id', (req, res) => {
+    const id = req.params.id;
+postDB
+    .getById(id)
+    .then(post => {
+        res.status(200).json(post)
+    })
+    .catch(error => {
+        res.status(400).json({
+            error: "couldnt get posts by id"
+        })
+    })
+})
+
+
+
+router.post('/', (req, res) => {
+    const postbod = req.body;
+
```

Comment:
We should really be checking if we're getting what the server requires on the post body, and then send back a response if we don't get what we need.